Formula/apib.rb

```diff
@@ -16,7 +16,7 @@ class Apib < Formula
     sha256 "c538b9389b1849704720383f03fb2fe4eaf145ecd01d7dca5fd08d95e0e961ba" => :mountain_lion
   end
 
-  depends_on :apr => :build
+  depends_on "apr-util" => :build
```

Comment:
There's quite a lot of that in the core that's wrong. I guess whenever it was changed nobody picked up on/remembered the `=> :build` was always a hack that needed discarding whenever Homebrew's `apr` was  used.